data/database.py

```diff
@@ -1901,6 +1903,51 @@ class RepoMirrorConfig(BaseModel):
     root_rule = ForeignKeyField(RepoMirrorRule)
 
 
+@unique
+class IndexStatus(IntEnum):
+    """
+    Possible statuses of manifest security scan progress.
+    """
+
+    MANIFEST_UNSUPPORTED = -2
+    FAILED = -1
```

Comment:
This is missing states like "Not yet indexed", which we need in case we want to start creating these rows optimistically

Reply:
We can easily add that in the future. Isn't it sufficient to simply check if the row exists as confirmation that it hasn't been indexed?

Reply:
Yeah, I guess. Let's leave it for now then.